src/Myddleware/RegleBundle/DataFixtures/ORM/LoadSolutionData.php

```diff
@@ -37,10 +37,8 @@ class LoadSolutionData implements FixtureInterface
 									array('name' => 'salesforce',		'active' => 1,'source' => 1,'target' => 1),
 									array('name' => 'prestashop',		'active' => 1,'source' => 1,'target' => 1),
 									array('name' => 'dolist',			'active' => 1,'source' => 1,'target' => 1),
-									array('name' => 'eventbrite',		'active' => 0,'source' => 1,'target' => 0),
```

Comment:
@Myddleware 

Are you sure to delete eventbrite and bittle solution ?

Reply:
Yes Dolyveen, we don't maintain these connectors anymore and nobody use it.
Thanks for your checking.